src/ggrc/services/suggest.py

```diff
@@ -42,21 +43,47 @@ def suggest():
     return mock_suggest()
 
   tokens = request.args.get("prefix", "").split()
+  initial_email = request.args.get("initialEmail", None)
   if tokens:
     person_client = client.PersonClient()
     entries = person_client.suggest_persons(tokens)
-    return make_suggest_result(entries)
+    return make_suggest_result(entries, initial_email)
   return make_suggest_result([])
 
 
-def make_suggest_result(entries):
+def _filter_emails(raw_suggest, initial_email):
+  """Filter by email suggested persons for suggesting persons are not in DB"""
```

Comment:
Let's rewrite the docstring for it to be more clear:
```suggestion
  """Filter out persons existing in the DB from suggested people."""
```
Also please provide the description for `raw_suggest` and `initial_email` parameters as well as for returning results.

Reply:
Done.